@thejrose1984 please review the failing tests

```text
Error: Missing required argument
│ 
│   on ../../main.tf line 110, in resource "aws_config_config_rule" "custom_policy_rules":
│  110:       content {
│ 
│ The argument "policy_runtime" is required, but no definition was found.
╵
╷
│ Error: Unsupported argument
│ 
│   on ../../main.tf line 112, in resource "aws_config_config_rule" "custom_policy_rules":
│  112:         policy_file_uri = each.value.policy_arn
│ 
│ An argument named "policy_file_uri" is not expected here.
╵}
    destroy.go:11: 
        	Error Trace:	destroy.go:11
        	            				panic.go:636
        	            				testing.go:1041
        	            				apply.go:15
        	            				examples_hipaa_test.go:36
        	Error:      	Received unexpected error:
        	            	FatalError{Underlying: error while running command: exit status 1; ╷
        	            	│ Error: Missing required argument
        	            	│ 
        	            	│   on ../../main.tf line 110, in resource "aws_config_config_rule" "custom_policy_rules":
        	            	│  110:       content {
        	            	│ 
        	            	│ The argument "policy_runtime" is required, but no definition was found.
        	            	╵
        	            	╷
        	            	│ Error: Unsupported argument
        	            	│ 
        	            	│   on ../../main.tf line 112, in resource "aws_config_config_rule" "custom_policy_rules":
        	            	│  112:         policy_file_uri = each.value.policy_arn
        	            	│ 
        	            	│ An argument named "policy_file_uri" is not expected here.
        	            	╵}
        	Test:       	TestExamplesHipaa
--- FAIL: TestExamplesHipaa (23.13s)
```

--- a/examples/custom-rules/variables.tf
+++ b/examples/custom-rules/variables.tf
@@ -84,7 +84,7 @@ variable "custom_policy_rules" {
   type = map(object({
     description      = string
     policy           = optional(string, null)
-    policy_arn       = optional(string, null)
+    policy_runtime   = optional(string, "guard-2.x.x")
     input_parameters = optional(any, {})
     scope = optional(object({
       compliance_resource_types = optional(list(string), [])
@@ -94,8 +94,9 @@ variable "custom_policy_rules" {
   }))
   default = {
     example_cfn_guard_rule = {
-      description = "CFN Guard rule for checking security group configurations."
-      policy      = <<-EOT
+      description    = "CFN Guard rule for checking security group configurations."
+      policy_runtime = "guard-2.x.x"
+      policy         = <<-EOT
         rule sg_restricted_ingress {
           aws_security_group.SecurityGroupIngress[*]  {
             cidr_ip != "0.0.0.0/0"

--- a/main.tf
+++ b/main.tf
@@ -102,14 +102,13 @@ resource "aws_config_config_rule" "custom_policy_rules" {
   description = each.value.description
 
   source {
-    owner             = "CUSTOM_POLICY"
-    source_identifier = "custom-policy"
+    owner = "CUSTOM_POLICY"
 
     dynamic "custom_policy_details" {
-      for_each = (each.value.policy != null || each.value.policy_arn != null) ? [1] : []
+      for_each = each.value.policy != null ? [1] : []
       content {
-        policy_text     = each.value.policy
-        policy_file_uri = each.value.policy_arn
+        policy_runtime = each.value.policy_runtime
+        policy_text    = each.value.policy
       }
     }
   }

--- a/test/src/examples_custom_rules_test.go
+++ b/test/src/examples_custom_rules_test.go
@@ -12,8 +12,8 @@ import (
 )
 
 func TestExamplesCustomRules(t *testing.T) {
-	rand.Seed(time.Now().UnixNano())
-	randID := strconv.Itoa(rand.Intn(100000))
+	rng := rand.New(rand.NewSource(time.Now().UnixNano()))
+	randID := strconv.Itoa(rng.Intn(100000))
 	attributes := []string{randID}
 
 	terraformOptions := &terraform.Options{

--- a/variables.tf
+++ b/variables.tf
@@ -254,15 +254,15 @@ variable "custom_lambda_rules" {
 variable "custom_policy_rules" {
   description = <<-DOC
     A map of custom policy-based Config rules (CFN Guard, etc.).
-    Uses Policy and PolicyArn for rule evaluation.
-    
+    Uses inline policy_text for rule evaluation.
+
     See the following for more information:
     https://docs.aws.amazon.com/config/latest/developerguide/evaluate-config_develop-rules_cfn-guard.html
   DOC
   type = map(object({
     description      = string
-    policy           = optional(string, null) # Policy text or CFN Guard rules
-    policy_arn       = optional(string, null) # S3 ARN to policy file
+    policy           = optional(string, null) # Inline CFN Guard policy text
+    policy_runtime   = optional(string, "guard-2.x.x")
     input_parameters = optional(any, {})
     scope = optional(object({
       compliance_resource_types = optional(list(string), [])

